opendbc/safety/safety/safety_honda.h

```diff
@@ -378,6 +387,10 @@ static safety_config honda_bosch_init(uint16_t param) {
     SET_RX_CHECKS(honda_common_alt_brake_rx_checks, ret);
   } else if (honda_bosch_radarless) {
     SET_RX_CHECKS(honda_bosch_radarless_rx_checks, ret);
+  } else if (honda_bosch_canfd && honda_alt_brake_msg) {
+    SET_RX_CHECKS(honda_common_alt_brake_rx_checks, ret);
+  } else if (honda_bosch_canfd) {
+    SET_RX_CHECKS(honda_bosch_canfd_rx_checks, ret);
```

Comment:
how does this work for the existing Pilots? seems like this should be a new platform definition if the RX/TX messages are different

Reply:
Existing pilots are Nidec and not covered by this